ecs/firehose.conf

```diff
@@ -0,0 +1,19 @@
+[SERVICE]
+    Grace 30
+
+[INPUT]
+    Name forward
+    unix_path /var/run/fluent.sock
+
+[FILTER]
+    Name record_modifier
+    Match *
+    Record ecs_cluster ${ECS_CLUSTER}
+    Record ecs_task_arn ${ECS_TASK_ARN}
+    Record ecs_task_definition ${ECS_TASK_DEFINITION}
+
+[OUTPUT]
+    Name   kinesis_firehose
+    Match  *
+    region ${FIREHOSE_REGION}
+    delivery_stream ${STREAM}
```

Comment:
should be DELIVERY_STREAM

Reply:
my doc was wrong, I just updated it. The name is supposed to match the plugin config key name